issuebot

```diff
@@ -131,6 +164,11 @@ lilac can't parse out the relevant package names, please handle manually.''')
     with file_lock(LILAC_LOCK):
       subprocess.check_output(['git', 'pull'], cwd=REPO)
 
+      for name in os.listdir(REPO):
+        if not os.path.isdir(os.path.join(REPO, name)):
+            continue
+        remove_repo_depends(name, packages)
```

Comment:
We'd set `changed = True` if we've made changes, or they won't be committed. Also we need to give different messages when the package itself has been removed but we are updating lilac.yaml.

Reply:
Seems a duplicate of one of previous review comments. Marked as resolved.